### DISCUSSION TITLE
How many parallel plots with an MP600, so I don't kill it?? I know +4x will kill a EVO-PRO980 ( New drive 4 hours OLD DEAD)

### DISCUSSION THREAD

**Discussion by room101-dev** [Chia-Network/chia-blockchain#6346] [Support]
I have burned up most of my NVME samsung evo-pro 980's plotting chia, so now I'm moving on to the MP600 as its in the chia-blog, they say its the 'best', 

So anybody here use one, and know the max? parallel staggered plots before it dies? Does it use the same controller as the EVO-PRO980?

With the EVO NVME is the random-access controller that over-heats and dies +90C out of the box when 'plotting', so I learned the hard way to keep fans on the NMVE's, and to modify the source so pause when temp goes over 70c

In the case of MP600, is there any life forms on planet-chia that has actually used one, or is this just another team-chai dev post that was fed by manufacturers, with no real use experience behind the blog post??

**Comment by room101-dev** [comment]
OK, first test first day, it died, 4 hours into process the NVME MP600 controller over-heated and died. DEAD, its done.

CLEARLY the MP600 is not up to task to plot chia plots.

**Comment by gromran** [comment]
> over-heated and died. DEAD, its done.

If you have no idea, then let it be

**Reply by temicus** [reply]
I plot with my 2TB MP600 already one month and have reached ca. 500 TB of write volume. I plot usually 6-8 plots at the same time but in different phases (usually for example start next 4 as soon the first 4 are passing the 31% progress line and go to the phase 2). But I think it also depends on the mainboard and cpu. And I have a good cooling system and its not getting hotter as 55-60'C. I have also MP510. This one is actually just a little but slowler but much chipper and also very stable with added cooler.

**Comment by BrandtH22** [comment]
@room101-dev , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
